src/utils.ts

```diff
@@ -991,10 +991,10 @@ export interface InterruptableAsyncInterval {
  *
  * @param fn - An async function to run on an interval, must accept a `callback` as its only parameter
  */
-export function makeInterruptableAsyncInterval(
+export function makeInterruptibleAsyncInterval(
```

Comment:
This import needs to be updated in `test/unit/utils.test.js` to fix CI

Reply:
oops thanks, fixed